packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -265,6 +275,60 @@ export function useBlockWrapperProps( props = {}, { __unstableIsHtml } = {} ) {
 		};
 	}, [ isNavigationMode, isHovered, setHovered ] );
 
+	useEffect( () => {
+		if ( isFullSiteEditingActive ) {
```

Comment:
How about an early return? We can save an indentation 😋 
```js
if ( ! isFullSiteEditingActive ) {
	return;
}
```

Reply:
Thats definite better. 😀